yggdrasil_decision_forests/utils/test.cc

```diff
@@ -41,6 +41,12 @@ std::string TmpDirectory() {
   return path;
 }
 
+#ifdef _WIN32
+
+bool IsPortAvailable(int port) {return true;}
```

Comment:
That's surprising, can you explain why there's always a port under windows? Or is this a form of bailing out of the test?

Reply:
This is a bailing out :).
Let me add a warning.